jdl/jhipster/default-application-options.js

```diff
@@ -227,6 +234,14 @@ function getDefaultConfigForNewApplication(customOptions = {}) {
   if (!options[PACKAGE_FOLDER] && options[PACKAGE_NAME]) {
     options[PACKAGE_FOLDER] = options[PACKAGE_NAME].replace(/\./g, '/');
   }
+  let skipClient = options[SKIP_CLIENT];
+  if (skipClient === undefined) {
+    skipClient = options[CLIENT_FRAMEWORK] === undefined || options[CLIENT_FRAMEWORK] === NO;
```

Comment:
Need more time to check the behavior, but the default is angular, not skipClient.

Reply:
ok you told me, but it fixes the issue 🙂

Reply:
ok you told me, but it fixes the issue 🙂